packages/datadog-instrumentations/src/helpers/register.js

```diff
@@ -19,6 +21,51 @@ const disabledInstrumentations = new Set(
 
 const loadChannel = channel('dd-trace:instrumentation:load')
 
+if (!dc.subscribe) {
+  dc.subscribe = (channel, cb) => {
+    dc.channel(channel).subscribe(cb)
+  }
+}
+if (!dc.unsubscribe) {
+  dc.unsubscribe = (channel, cb) => {
+    if (dc.channel(channel).hasSubscribers) {
+      dc.channel(channel).unsubscribe(cb)
+    }
+  }
+}
+
+dc.subscribe('dd-trace-esbuild', (payload) => {
```

Comment:
Is this really specific to esbuild? Also, why not have this in another file like before?

Reply:
I suppose it's more specific to bundling. Went with `dd-trace:bundler:load`.

Reply:
Also, this has been moved to a new file.